src/common/services/api/profile.service.js

```diff
@@ -176,7 +177,7 @@ class Profile {
 
 
   updateMailingAddress(mailingAddress){
-    let mailingAddressCopy = Object.assign({}, mailingAddress);
```

Comment:
This change was because phantomjs seemed to not know what `Object.assign` was and a test was failing. Maybe it needs a polyfill but using lodash was a easy fix. Idk how that test passed before.